docs/root/start/sandboxes/route_mirroring.rst

```diff
@@ -0,0 +1,126 @@
+.. _install_sandboxes_route_mirroring:
+
+Route mirroring policies
+========================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+This simple example demonstrates Envoy's request mirroring capability using
+`request mirror policies <https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-msg-config-route-v3-routeaction-requestmirrorpolicy>`__.
+
+Incoming requests are received by ``front-envoy`` Envoy proxy.
+
+The ``front-envoy.yaml`` envoy configuration configures two routes::
```

Comment:
rather use a `literalinclude` to include the config (if you grep in the docs for `literalinclude` you will find examples)